simespresso/io_QE/tests/test_start_espresso.py

```diff
@@ -0,0 +1,24 @@
+
+import unittest
+import os
+from simespresso.io_QE import espresso_class
+
+
+class OutcomesTest(unittest.TestCase):
```

Comment:
> as for start_qe - shouldn`t that be in the espresso_class, since other simphony modules will be wanting to call it ?

users of simphoyn-espresso-dft will be using the to-be-implemented concrete class of `ABCModelingEngine`